src/code-templates/libraries/jwt/jwt.ts

```diff
@@ -0,0 +1,31 @@
+import jwt from 'jsonwebtoken';
+
+export interface JWTOptions {
+  secret: string;
```

Comment:
Important: Put todo or a comment with warning that sync JWT flow is not considered secure and one should prefer the async JWT flow with JWKS (when reading the public key from some issuer)

In our next iteration, let's use JWKS library? We can chat on this